### PR TITLE
Expose the commit dates

### PR DESCRIPTION
**Pull request by stephenc**
I need this for SCM API stuff

### CODE REVIEW

File: src/main/java/org/kohsuke/github/GHCommit.java [hunk 1]
```diff
@@ -42,11 +42,19 @@ public GitUser getAuthor() {
             return author;
         }
 
+        public Date getAuthoredDate() {
+            return GitHub.parseDate(author.date);
```
Comment (KostyaSha) [inline src/main/java/org/kohsuke/github/GHCommit.java]: Why `authored`? It's `AuthorDate`

File: src/main/java/org/kohsuke/github/GHCommit.java [hunk 2]
```diff
@@ -42,11 +42,19 @@ public GitUser getAuthor() {
...
+        }
+
         @WithBridgeMethods(value = GHAuthor.class, castRequired = true)
         public GitUser getCommitter() {
             return committer;
         }
 
+        public Date getCommitDate() {
+            return GitHub.parseDate(author.date);
```
Comment (KostyaSha) [inline src/main/java/org/kohsuke/github/GHCommit.java]: So what is the difference of this two methods? Copy-paste instead `commit.date`?

**Comment by KostyaSha** [comment]
@kohsuke did you review it or blindly merged and released?

**Comment by KostyaSha** [comment]
@stephenc ?

**Comment by kohsuke** [comment]
Good catch! I did look at the change but I missed it.

I'll fix this and push out a new release.

**Comment by kohsuke** [comment]
I'll leave the `getAuthoredDate` as is. I trust Stephen's English than mine and yours.

**Comment by stephenc** [comment]
Good catch on the copy and paste.

So I went with "commitDate" rather than "committerDate" and "authoredDate" vs "authorDate" as these are the dates of the respective actions. In a sense GitHub conflates the action with the person, and it just sounded wrong to my ear to say "getCommitterDate"... what's the committer date... does that method arrange for flowers a movie and a meal with the committer... ok, perhaps the committer deserves that as a reward for their work... that is more the date of the commit... closest analogy for author I could find was authored

**Comment by KostyaSha** [comment]
Question not in english, it not a novel. When you know API you can use library easily, otherwise you should scroll code/methods to get idea what they return. Also why you can't call getter().field yourself and need this proxy method?

**Pull request merged**